x-pack/plugins/apm/common/utils/formatters/duration.ts

```diff
@@ -185,7 +190,10 @@ export function asTransactionRate(value: Maybe<number>) {
 export function asExactTransactionRate(value: number, unit: ThroughputUnit) {
   return i18n.translate('xpack.apm.exactTransactionRateLabel', {
     defaultMessage: `{value} { unit, select, minute {tpm} other {tps} }`,
-    values: { value: asDecimalOrInteger(value), unit },
+    values: {
+      value: value >= 1 ? asDecimalOrInteger(value) : asPreciseDecimal(value),
+      unit,
+    },
```

Comment:
Can you show examples when the extra decimals are needed?

Reply:
<img width="849" alt="Screen Shot 2021-09-20 at 13 10 34" src="https://user-images.githubusercontent.com/55978943/134186550-52d047b5-21ae-4bde-a8f7-6ddcca937e98.png">

Reply:
Okay, I'm actually leaning towards always keeping the rate in tpm. Then we don't have this problem, right?

Reply:
no. is it something you'd like to do on this PR?